### ISSUE TITLE
Crash when trying to add reputation reward in my 1.16.5 modpack

### ISSUE THREAD

**Issue by vizthex123** [lorddusk/HQM#627]
I clicked the "edit reputation reward" icon, and it crashed. Not sure why.

Crash report: [crash-2021-11-09_22.00.15-client.txt](https://github.com/lorddusk/HQM/files/7509739/crash-2021-11-09_22.00.15-client.txt)
Game log: [latest.log](https://github.com/lorddusk/HQM/files/7509740/latest.log)

**Comment by kirderf1** [comment]
Tested it with the same forge and HQM versions, but was not able to reproduce the issue. I have three ideas:

- Mod incompatibility or just plain issue caused by another mod.
  There are a lot of other mods present based on your crashlog, and I was unable to reproduce the issue with just forge and HQM. As such, some other mod could be a suspect, even if it doesn't seem too likely with the nature of the crash.
- Corrupted HQM jar.
  The issue is a failure to load a class (+ one more class-loading failure in the log) in HQM. As such, a possible cause could be that the jar file was corrupted during or after download.
- Outdated java version.
  I happened to notice a line in the log `LEGACY JDK DETECTED, SECURED JAR HANDLING DISABLED`, and that you're using java version 1.8.0_51, which is a fairly outdated version of java 8. I'm not sure how likely this is to be a cause, but it seems like a possibility.
